disco_aws_automation/disco_storage.py

```diff
@@ -165,17 +165,15 @@ def configure_storage(self,
                           extra_space=None,
                           extra_disk=None,
                           iops=None,
-                          ephemeral_disk_count=0,
-                          map_snapshot=True):
-        '''Alter block device to destroy the volume on termination and add any extra space'''
+                          ephemeral_disk_count=0):
+        ''' Cofigures block device mapping for root partition, extra disk if it's used,
+        and the ephemeral discks. Note that EBS volume mapping is no longer being created
+        here. The logic for attaching EBS volume has been moved to /etc/asiaq/init/ebs-start.sh '''
         # Pylint thinks this function has too many local variables
         # pylint: disable=R0914
 
-        # We map disk names starting at /dev/sda, but aws shifts everything after /dev/sda
-        # to the right four characters, i.e /dev/sdb becomes /dev/sdf, /dev/sdc becomes /dev/sde
-        # and so on.
-        # TODO  Figure out how to stop this from happening
-        disk_names = ['/dev/sd' + chr(ord('a') + i) for i in range(0, 26)]
+        # disk names only go up to /dev/sdy. /dev/sdz is reserved for EBS volume
+        disk_names = ['/dev/sd' + chr(ord('a') + i) for i in range(0, 25)]
```

Comment:
@pchand-amplify : ok they have not put that into pipeline. that's why i have not seen it. So the question is if they have a lot of temp file. should they use the ephemeral storage by using a larger instances instead of extra disk? Disco used extra disk for temp space until we find out we can mount ephemeral disk (instance SSD) and it is way more faster.

But if that's a pain for them to change. then i guess we need to make sure the volume finding for extra disk and snapshot volume not causing problem now due to large instances have almost no block device id for both of them.